extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -1,73 +1,253 @@
+        <span style="padding-left: 3px;">Bold</span>
+      </div>
+      <div class="input-tag" ng-if="$ctrl.displayFontStyles">
+        <select ng-model="$ctrl.fabricjsOptions.fontFamily" ng-options="font for font in $ctrl.fontFamily"
+                ng-change="$ctrl.onFontChange()">
+        </select>
+      </div>
+      <div class="input-tag">
+        <select ng-model="$ctrl.fabricjsOptions.size" ng-options="size for size in $ctrl.sizes"
+                ng-change="$ctrl.onSizeChange()">
+        </select>
+      </div>
+      <div class="input-tag" ng-if="$ctrl.isDrawModePencil()" style="color: #f00101;">
+        Click the pencil button to exit free drawing mode.
+      </div>
+      <div class="input-tag" ng-if="$ctrl.isDrawModePolygon()" style="color: #f00101;">
+        Double click to complete the polygon.
+      </div>
+    </div>
   </div>
   <div ng-if="$ctrl.isDiagramSaved()">
     <div class="diagram-editor-saved-image-container">
-      <i class="material-icons diagram-editor-edit"
-         ng-click="$ctrl.continueDiagramEditing()"
+      <i class="material-icons diagram-editor-edit edit-icon" ng-click="$ctrl.continueDiagramEditing()"
          title="Continue editing the diagram">
-         &#xE254;
+        &#xE254;
       </i>
       <img ng-src="<[$ctrl.data.savedSVGUrl]>" ng-style="$ctrl.svgContainerStyle">
     </div>
   </div>
   <div ng-if="$ctrl.isDiagramCreated()" class="diagram-editor-save-box">
-    <button type="button"
-            class="btn btn-success"
-            ng-click="$ctrl.saveSVGFile()"
-            ng-disabled="$ctrl.isUserDrawing()">
+    <button type="button" class="btn btn-success" ng-click="$ctrl.saveSVGFile()" ng-disabled="$ctrl.isUserDrawing()">
       Use This Image
     </button>
   </div>
 </div>
+<style>
+  .diagram-editor-saved-image-container {
+    position: relative;
+  }
 
-<style type="text/css">
-  .diagram-editor-edit {
-    color: #000000;
+  .diagram-editor-saved-image-container .edit-icon {
+    color: black;
     position: absolute;
     right: 6px;
-    text-shadow: 0px 0px 1px #ffffff;
+    text-shadow: 0px 0px 1px #fff;
     top: 6px;
-    cursor: pointer;
   }
 
-  .diagram-editor-save-box {
-    margin-top: 6px;
-    text-align: right;
+  .svg-editor {
+    background-color: #ffffff;
+    min-height: 400px;
+    position: relative;
+  }
+
+  .svg-editor>* {
+    position: absolute
   }
 
-  .diagram-editor-saved-image-container {
+  .svg-editor,
+  .svg-editor * {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box
+  }
+
+  .svg-editor .fabric-canvas {
+    border: 1px solid #555555;
+    bottom: 31px;
+    cursor: default;
+    left: 62px;
+    right: 0;
+    top: 0;
+  }
+
+  .svg-editor .horz-toolbar {
+    background-color: #ffffff;
+    border: 1px solid #555555;
+    height: 31px;
+    left: 62px;
+    right: 0;
+    bottom: 0;
+  }
+
+  .svg-editor .horz-toolbar .input-tag {
```

Comment:
```suggestion
  .svg-editor .horizontal-toolbar .input-tag {
```
I think we should use the full word for this.

Reply:
Done.